server/src/packages/rwcds-form/rwcds-form.service.ts

```diff
@@ -106,6 +106,12 @@ export class RwcdsFormService {
   }
 
   async find(id) {
+    try {
+
```

Comment:
wait why is there an empty try block?

Reply:
my bad. I didn't move the `get` into the block. fixed.